documentation/configure.markdown

```diff
@@ -260,11 +260,23 @@ PuppetDB stores its data in PostgreSQL.
 ### Using PostgreSQL
 
 Before using the PostgreSQL backend, you must set up a PostgreSQL
-server, ensure that it will accept incoming connections, create a user
-for PuppetDB to use when connecting, and create a database for
-PuppetDB. Completely configuring PostgreSQL is beyond the scope of
-this guide, but if you are logged in as root on a running PostgreSQL
-server, you can create a user and database as follows:
+server. Note that users installing PuppetDB via [the module][module]
```

Comment:
I thought when using a reference like module you had to define it this way. Looking around in the file we use parens an brackets fairly inconsistently. I like the parens better anyway, so I'll switch it.